core/src/display_object/edit_text.rs

```diff
@@ -1032,6 +1014,43 @@ impl<'gc> EditText<'gc> {
         context.transform_stack.pop();
     }
 
+    fn render_selection(
+        self,
+        context: &mut RenderContext<'_, 'gc>,
+        x: Twips,
+        width: Twips,
+        height: Twips,
+    ) {
+        let selection_box = context.transform_stack.transform().matrix
+            * Matrix::create_box(
+                width.to_pixels() as f32,
+                height.to_pixels() as f32,
+                0.0,
+                x,
+                Twips::ZERO,
+            );
+        context.commands.draw_rect(Color::BLACK, selection_box);
```

Comment:
Is this really always `BLACK`?
I see this was similarly hardcoded before at the call site, but if it can change, perhaps adding a parameter and keeping the value there, or just adding a TODO comment, would be nice.

Reply:
Otherwise LGTM, thank you!

Reply:
It's a good question. It seems like black is the default and it cannot be changed at all. I could find only the following.
* https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flashx/textLayout/edit/SelectionManager.html#focusedSelectionFormat
* https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flashx/textLayout/elements/Configuration.html#focusedSelectionFormat

From what I understand, it's a library which somehow implements a custom selection color on its own (https://github.com/apache/flex-tlf/blob/master/textLayout/src/flashx/textLayout/container/ContainerController.as#L2901).

Reply:
Ah, I see. But all of these are for `TextLayout`, and this PR is only about `EditText`, right?

Reply:
If I understand it correctly, it's for `TLFTextField`, which is from an external library included inside the SWF (so Ruffle and flashplayer do not implement it at all). It uses some mechanisms (sprites, etc.) to model the field so it looks like `TextField` but has more features (like a custom selection color).

Ruffle only implements `TextField` (internally as `EditText`), and from my reconnaissance it seems like the selection color is always black and cannot be changed. The internet always redirects to `TLFTextField` when someone wants to change the selection color. It's also confirmed by my testing, where custom fonts, colors, colorTransforms, etc. do not change the behavior: the selection is always black.